packaging/images/microshift/Dockerfile

```diff
@@ -29,7 +29,12 @@ RUN microdnf install -y \
     && microdnf clean all
 COPY --from=builder /opt/app-root/src/github.com/redhat-et/microshift/_output/bin/linux_$ARCH/microshift /usr/bin/microshift
 
-ENTRYPOINT ["/usr/bin/microshift"]
+RUN mkdir -p /root/crio.conf.d
+
+COPY packaging/crio.conf.d/microshift.conf /root/crio.conf.d/microshift.conf
+COPY packaging/images/microshift/entrypoint.sh /root/entrypoint.sh
+
+ENTRYPOINT ["/root/entrypoint.sh"]
 CMD ["run"]
```

Comment:
@mangelajo you should probably remove ```CMD ["run"]```. I don't think it'll hurt anything, but it's passing "run" as an argument to entrypoint.sh, which would be ignored